src/components/views/RoleView/RoleView.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from 'react';
+import Box from 'components/Box';
+import { Role, RoleData } from 'modules/roles/types';
+import { UserData } from 'modules/users/types';
+import { useCollectionData } from 'modules/data';
+import { ROLES } from 'utils/collectionNames';
+
+const RoleView = (props: RoleViewProps) => {
+  const { role, users } = props;
```

Comment:
How come u didn't deconstruct in the param list?

```suggestion
const RoleView = ({ role, users }: RoleViewProps) => {
```